app/controllers/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t("signup_suc")
+      redirect_to @user
+    else
+      flash[:danger] = t("signup_failed")
```

Comment:
```suggestion
      flash.now[:danger] = t "signup_failed"
```
phân biệt flash và flash.now cho chị nghe nhé

Reply:
flash hiển thị ở trang tiếp theo còn flash.now hiển thị ở trang hiện tại.